docs/rules/no-shadow.md

```diff
@@ -50,12 +50,12 @@ This rule takes one option, an object, with properties `"builtinGlobals"`, `"hoi
 }
 ```
 
-### `builtinGlobals`
+### builtinGlobals
```

Comment:
Am suggesting that the example sentence use code style to display an option that is consistent with the eslint comment in the code block. Most style guides minimize punctuation and formatting in headings to maximize scanning for relevant content (in this case, option) in the page. It also leans toward possibility of a table of contents.